draft-ietf-git-using-github.md

```diff
@@ -23,6 +23,13 @@ author:
    email: barbara.stark@att.com
 
 informative:
+  GLOSSARY:
+    title: "GitHub glossary"
+    date: 2020-03
+    target: "https://help.github.com/en/github/getting-started-with-github/github-glossary"
```

Comment:
Are we concerned about this link not aging well? (I wonder if we should, in parallel, try to move the contents of https://ietf-gitwg.github.io to some place more stable and under control of the IETF.)

Reply:
Yeah, but I'm not sure that our own copy of this is guaranteed to be any better.  The GitHub link is nice in the sense that it concentrates on terminology.

Reply:
Yep, no argument there. It certainly has better content.